index-test.js

```diff
@@ -39,33 +39,19 @@ test("form submits expected result", () => {
 
 */
 
-setTimeout(() => {
-
-  test("deleteItem() function should remove an item from the list", () => {
-    const deleteBtn = document.querySelectorAll('button');
-    const movieDiv = document.querySelectorAll('.movie-name');
-    const movietitle = document.querySelectorAll('.movie-title');
-    if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
-    deleteBtn[0].click(); //When the first delete button is clicked...
-    const result = movieDiv.includes(movietitle[0])
-    const expected = false
-    equal (result, expected);
-    deleteBtn[0].click();
-  })
-  }, 5000);
-  
-
-setTimeout(() => { //Allows time for the elements to appear on the page as they're not present onload.
-test("deleteItem() function should remove an item from the list", () => {
-  const movieDiv = document.querySelectorAll('.movie-name') //Outer container
-  const movieTitle = document.querySelectorAll('.movie-title') //Input value
-  const deleteBtn = document.querySelectorAll('button') //delete button
-  if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
-  let result = deleteBtn[0].click(); //When the first delete button is clicked...
-  const expected = deleteItem(movieDiv.contains(movieTitle[0]) == false); //...the first input will no longer be contained in the div.
-  // const expected = false;
-  equal (result, expected); //When result takes place, expected is what we should see.
-  })
-  }, 5000);
-
-//Remove the delete button's class name. (Added in for test purposes)
+
+
+test("Delete button removes items from the list", () => {
+document.querySelector('button#new-list').click(); //Click the TV button to generate new list
+const input = document.querySelector('input');
+input.value = "test"; //Add input value.
+document.querySelector('button#add-title').click(); //Click the 'plus' button to submit input.
```

Comment:
Great commenting throughout, super clear!

Reply:
Really enjoyed reading the click on the plus button 🥳 ✨